src/fidesctl/cli/utils.py

```diff
@@ -40,12 +40,13 @@ def check_server(cli_version: str, server_url: str) -> None:
         raise SystemExit(1)
 
     server_version = health_response.json()["version"]
-    if str(server_version) == str(cli_version):
-        echo_green(
-            "Server is reachable and the client/server application versions match."
-        )
+    if str(server_version).replace(".dirty", "", 1) == str(cli_version):
+        if not quiet:
+            echo_green(
+                "Server is reachable and the client/server application versions match."
+            )
     else:
-        echo_red(
+        echo_yellow(
```

Comment:
I realize red "feels" like a strong response, but this is still the error state for this command. We don't know for sure that it's fine to have mismatched CLI and server versions, because we don't know for sure how mismatched the versions will be. I prefer to keep this red, and to work through a more robust logging/output solution on a dedicated issue.

Reply:
mm okay yeah that makes sense. will revert and then make a new issue for more robust logging